src/components/CertificateViewer.js

```diff
@@ -9,47 +9,66 @@ import { MultiTabs } from "./DecentralisedTemplateRenderer/MultiTabs";
 import { DocumentStatus } from "./DocumentStatus";
 import { DocumentUtility } from "./DocumentUtility";
 import { AssetManagementContainer } from "./AssetManagementPanel/AssetManagementContainer";
+import { getData } from "@govtechsg/open-attestation";
+import { Tab } from "react-bootstrap";
+import { AttachmentLink } from "./UI/AttachmentLink";
 
 export const CertificateViewer = (props) => {
   const { document } = props;
   const tokenRegistryAddress = getTokenRegistryAddress(document);
-  const [templates, setTemplates] = useState([]);
+  const [, setTemplates] = useState([]);
   const [selectedTemplate, setSelectedTemplate] = useState("");
+  const originalData = getData(document);
+  const attachments = originalData?.attachments;
 
   const updateTemplates = useCallback((templates) => {
     setTemplates(templates);
     setSelectedTemplate(templates[0].id);
   }, []);
 
   const validCertificateContent = (
-    <>
-      <section className="bg-blue-lighter no-print">
+    <Tab.Container defaultActiveKey="tab-document">
+      <div className="bg-blue-lighter no-print">
         <DocumentStatus verificationStatus={props.verificationStatus} />
         {tokenRegistryAddress && <AssetManagementContainer document={document} />}
-        <MultiTabs
-          templates={templates}
-          selectedTemplate={selectedTemplate}
-          onSelectTemplate={(selectedTemplate) => setSelectedTemplate(selectedTemplate)}
-          tokenRegistryAddress={tokenRegistryAddress}
-        />
-      </section>
-      <section>
-        <DocumentUtility className="no-print" document={document} handleSharingToggle={props.handleSharingToggle} />
-        <div className="bg-white py-3">
-          <DecentralisedRendererContainer rawDocument={document} updateTemplates={updateTemplates} />
-        </div>
-      </section>
+        <MultiTabs tokenRegistryAddress={tokenRegistryAddress} attachments={attachments} />
+      </div>
+      <div className="bg-white">
+        <Tab.Content className="py-4">
+          <Tab.Pane eventKey="tab-document">
+            <DocumentUtility className="no-print" document={document} handleSharingToggle={props.handleSharingToggle} />
+            <DecentralisedRendererContainer
+              rawDocument={document}
+              updateTemplates={updateTemplates}
+              selectedTemplate={selectedTemplate}
+            />
+          </Tab.Pane>
+          {attachments && (
```

Comment:
Better to have a property `hasAttachments` and then check also it doesnt have length 0.